feat: Git provider integration at organization level PLUTO-535 (#1706)
codacy/docs: docs/repositories-configure/integrations/gitlab-integration.md

## Enabling the GitLab integration {: id="enabling"}

> To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, the integration is already enabled by default with the [settings defined for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).

Author: [Currently](https://docs.codacy.com/repositories-configure/integrations/github-integration/#enabling), when you add a new repository, the integration is already enabled by default.
From now, this integration (that Codacy enables by default) will use the settings defined for the organization on the new Git provider page.

Would it help to replace "with" with "using"? 🤔 

```suggestion
To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, the integration is already enabled by default using the [settings defined for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).
```

> To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, the integration is already enabled by default with the [settings defined for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).

Reviewer: In this case I think it can simplify a lot to get rid of the passive voice. For example:

```suggestion
To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, Codacy integrates with GitLab using the [default settings for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).
```

> To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, the integration is already enabled by default with the [settings defined for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).

Author: For consistency, I would keep "enable the integration" instead of "integrate". WDYT, @prcr?
@inesgomas Is this sentence more clear?

```suggestion
To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, Codacy enables the integration using the [default settings for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).
```

> To enable the GitLab integration, open your repository **Settings**, tab **Integrations**. When you add a new repository, the integration is already enabled by default with the [settings defined for your organization](../../organizations/configuring-default-git-provider-integration-settings.md).

Reviewer 2: YES 🙌
